React-Homeworks/src/components/Header/Header.jsx

```diff
@@ -1,21 +1,21 @@
-import React, { Component } from "react";
 import NavBar from "./NavBar/NavBar";
 import Logo from "../Logo/Logo";
-import "./Header.css";
 import Cart from "./Cart/Cart";
+import "./Header.css";
+import { useCart } from "../../Context/CartContext";
+
+const Header = () => {
+  const { cartQuantity } = useCart();
```

Comment:
Why do you use context here? It looks the better place for this is Cart component